algolia/algoliasearch-sfcc-b2c
# feat: Maintain Cartridge Version via Github action and displaying it on BM

# Automatic Cartridge version change

Simply we are changing package.json version and version.isml file for each release by using GitHub action.

## Changes

- Automatic Package.json version change
- Automatic version change on BM
- To differentiate if it is develop branch or a stable release

## How to test

- Run Github action via UI
- Check [this fork and logs of Github Action](https://github.com/htuzel/algoliasearch-sfcc-b2c/actions)
